src/game/Utils/Cinematics.cc

```diff
@@ -30,8 +30,8 @@ struct SMKFLIC
 {
 	HWFILE    hFileHandle;
 	Smack*    SmackerObject;
-        CHAR8     SmackerStatus;
-        SDL_Surface*    SmackBuffer;
+	CHAR8     SmackerStatus;
+	SDL_Surface*    SmackBuffer;
 	UINT32    uiFlags;
 	UINT32    uiLeft;
 	UINT32    uiTop;
```

Comment:
I don't want to do that if it takes more than four spaces. IMO it simply adds more clutter and takes a lot of effort.